critiquebrainz/ws/review/views.py

```diff
@@ -283,10 +283,10 @@ def review_list_handler():
                 "id": "CC BY-NC-SA 3.0",
                 "info_url": "https:\/\/creativecommons.org\/licenses\/by-nc-sa\/3.0\/"
               },
+              "rating": 0,
```

Comment:
`rating` values shouldn't be listed there if they are no longer returned though. Just replace them with `popularity`.